README.md

```diff
@@ -1 +1,5 @@
-New Project
+# FCC Test Suite Template
+
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+
+You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
```

Comment:
Kindly give the project a  **Project title** and a proper **Project description** in the README section to make your work look more professional. 😄